.setup/INSTALL_SUBMITTY_HELPER_BIN.sh

```diff
@@ -52,16 +52,18 @@ mkdir -p ${SUBMITTY_INSTALL_DIR}/sbin
 chown root:root ${SUBMITTY_INSTALL_DIR}/sbin
 chmod 755 ${SUBMITTY_INSTALL_DIR}/sbin
 
-mkdir -p ${SUBMITTY_INSTALL_DIR}/sbin/autograder
+mkdir -p ${SUBMITTY_INSTALL_DIR}/autograder
 mkdir -p ${SUBMITTY_INSTALL_DIR}/sbin/shipper_utils
 
 # copy all of the files
 rsync -rtz  ${SUBMITTY_REPOSITORY}/sbin/*   ${SUBMITTY_INSTALL_DIR}/sbin/
-rsync -rtz  ${SUBMITTY_REPOSITORY}/autograder/autograder/* ${SUBMITTY_INSTALL_DIR}/sbin/autograder/
+rsync -rtz  ${SUBMITTY_REPOSITORY}/autograder/* ${SUBMITTY_INSTALL_DIR}/autograder/
 
 # most of the scripts should be root only
 find ${SUBMITTY_INSTALL_DIR}/sbin -type f -exec chown root:root {} \;
 find ${SUBMITTY_INSTALL_DIR}/sbin -type f -exec chmod 500 {} \;
+find ${SUBMITTY_INSTALL_DIR}/autograder -type f -exec chown root:root {} \;
+find ${SUBMITTY_INSTALL_DIR}/autograder -type f -exec chmod 500 {} \;
```

Comment:
These lines can be deleted as these permissions are overwritten by the chown/chmod you added down below.

Reply:
Fixed in recent commit.